manager/src/test/java/com/redhat/service/bridge/manager/workers/resources/AbstractWorkerTest.java

```diff
@@ -53,4 +77,49 @@ void workIsCompletedWhenResourceIsNotFound() {
         verify(workManager).complete(work);
     }
 
+    @Test
+    @Transactional
+    void workIsCompletedWhenMaxRetriedExceeded() {
+        Bridge bridge = new Bridge(TestConstants.DEFAULT_BRIDGE_NAME);
```

Comment:
@manstis Take a look at `com.redhat.service.bridge.manager.utils.Fixtures`, I think it gives you most of what you need here already.